ui/apps/everest/src/pages/db-cluster-details/db-cluster-details.tsx

```diff
@@ -76,99 +67,95 @@ export const DbClusterDetails = () => {
+          <Box
+            sx={{
+              display: 'flex',
+              justifyContent: 'space-between',
+              flex: '1 0 auto',
+              alignItems: 'center',
+            }}
+          >
+            <StatusField
+              dataTestId={dbClusterName}
+              status={dbCluster?.status?.status || DbClusterStatus.unknown}
+              statusMap={DB_CLUSTER_STATUS_TO_BASE_STATUS}
+            >
+              {beautifyDbClusterStatus(
+                dbCluster?.status?.status || DbClusterStatus.unknown
+              )}
+            </StatusField>
+            <DbActions isDetailView={true} dbCluster={dbCluster} />
+          </Box>
         </Box>
-      </Box>
-      <Box
-        sx={{
-          borderBottom: 1,
-          borderColor: 'divider',
-          mb: 1,
-        }}
-      >
-        <Tabs
-          value={currentTab}
-          variant="scrollable"
-          allowScrollButtonsMobile
-          aria-label="nav tabs"
+        <Box
+          sx={{
+            borderBottom: 1,
+            borderColor: 'divider',
+            mb: 1,
+          }}
         >
-          {Object.keys(DBClusterDetailsTabs).map((item) => (
-            <Tab
-              // @ts-ignore
-              label={Messages[item]}
-              // @ts-ignore
-              key={DBClusterDetailsTabs[item]}
-              // @ts-ignore
-              value={DBClusterDetailsTabs[item]}
-              // @ts-ignore
-              to={DBClusterDetailsTabs[item]}
-              component={Link}
-              data-testid={`${
-                DBClusterDetailsTabs[item as DBClusterDetailsTabs]
-              }`}
-            />
-          ))}
-        </Tabs>
+          <Tabs
+            value={currentTab}
+            variant="scrollable"
+            allowScrollButtonsMobile
+            aria-label="nav tabs"
+          >
+            {Object.keys(DBClusterDetailsTabs).map((item) => (
+              <Tab
+                // @ts-ignore
+                label={Messages[item]}
+                // @ts-ignore
+                key={DBClusterDetailsTabs[item]}
+                // @ts-ignore
+                value={DBClusterDetailsTabs[item]}
+                // @ts-ignore
+                to={DBClusterDetailsTabs[item]}
+                component={Link}
+                data-testid={`${
+                  DBClusterDetailsTabs[item as DBClusterDetailsTabs]
+                }`}
+              />
+            ))}
+          </Tabs>
+        </Box>
+        {dbCluster.status?.status === DbClusterStatus.restoring && (
+          <Alert severity="warning" sx={{ my: 1 }}>
+            {Messages.restoringDb}
+          </Alert>
+        )}
+        <Outlet />
       </Box>
-      {dbCluster.status?.status === DbClusterStatus.restoring && (
-        <Alert severity="warning" sx={{ my: 1 }}>
-          {Messages.restoringDb}
-        </Alert>
+      {clusterDeleted && (
+        <Dialog open>
```

Comment:
just a small suggestion is to put the dialog in a separate place, so that the component does not grow

Reply:
good one !